blog-service/2024-08-21-search.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Behaviour Change for `count` Operator (Search)
+image: https://help.sumologic.com/img/sumo-square.png
+keywords:
+  - count
+  - search-operator
+  - log-search
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We're happy to announce a change in the behavior of the `count` operator. Previously, the `count` operator was supported by the `Int` data type, which could result in a negative count value when exceeding the maximum integer value. To address this, we are changing the data type of the `count` operator from `Int` to `Long` data type to support the handling of large numeric values. 
+
+This change will apply to all new queries. Any existing queries as part of various content types - Log Search, [Scheduled Search](/docs/manage/scheduled-views/scheduled-views-best-practices/#upgrade-your-scheduled-views-to-avoid-negative-count-results), [Lookup Table](/docs/search/lookup-tables/manage-update-lookup-tables/#upgrade-the-lookup-table-to-avoid-negative-count-results), and Saved Search will also seamlessly transition to show `Long` values in case of `Int` overflow.
```

Comment:
hyperlink is for Scheduled View. so please add Scheduled View after scheduled Search and link this particular URL with scheduled view.